frontend/src/components/TrackingFieldsContent.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import {Descriptions} from "antd";
+import {withUser} from "../utils/withItem";
+
+export const TrackingFieldsContent = ({usersByID, entity}) => {
```

Comment:
Why not `connect()` this component rather than pass it `usersById` through the parent component? You're repeating the `usersById={usersById}` multiple times.